field_records/ubna_2023.csv

```diff
@@ -31,13 +31,19 @@ Date and Time deployed (local) , Date and Time recovered (local) , AudioMoth # ,
 2023-06-22T13:49:00 , 2023-06-28T13:33:00 , F , 011 , Central Pond , 47° 39' 17.502'' N , 122° 18' 3.060'' W , 48000 , Medium , None , None , 4.186 , 3.797 , CT , MB , CT , recover-20230628 , Panasonic Batteries Daytime 0:00-24:00 UTC
 2023-06-28T13:03:00 , 2023-07-05T13:20:00 , A , 007 , Telephone Field , 47° 39' 30.480'' N , 122° 17' 18.240'' W , 48000 , Medium , None , None , 4.140 , 3.760 , CT , MB , CT , recover-20230705 , Ikea Batteries Nighttime 3:00-13:30 UTC Checked configurations on laptop
 2023-06-28T13:03:00 , 2023-07-05T13:20:00 , B , 008 , Telephone Field , 47° 39' 30.480'' N , 122° 17' 18.240'' W , 192000 , Medium , None , None , 4.233 , 3.657 , CT , MB , CT , recover-20230705 , Ikea Batteries Daytime 0:00-24:00 UTC Checked configurations on laptop
-2023-06-28T13:22:00 , 2023-07-05T13:37:00 , G , 006 , Foliage , 47° 39' 18.750'' N , 122° 17' 26.940'' W , 192000 , Medium , None , None , 4.202 , 3.743 , CT , MB , CT , recover-20230705 , Panasonic Batteries Nighttime 3:00-13:30 UTC New AudioMoth switched out for C
+2023-06-28T13:22:00 , 2023-07-05T13:37:00 , G , 006 , Foliage , 47° 39' 18.750'' N , 122° 17' 26.940'' W , 192000 , Medium , None , None , 4.202 , 3.743 , CT , MB , CT , recover-20230705 , Panasonic Batteries Nighttime 3:00-13:30 UTC New AudioMoth switched out for C Reconfigured Set to 03:09-13:11
```

Comment:
```suggestion
2023-06-28T13:22:00 , 2023-07-05T13:37:00 , G , 006 , Foliage , 47° 39' 18.750'' N , 122° 17' 26.940'' W , 192000 , Medium , None , None , 4.202 , 3.743 , CT , MB , CT , recover-20230705 , Panasonic Batteries Nighttime 3:09-13:11 UTC; New AudioMoth switched out for C; Reconfigured Recording Period

Reply:
Rather than keep the "3:00-13:30 UTC", I think it would be better to just update that and say we reconfigured.

Also have you considered putting separators (like the semicolons I put in here) for each separate note?